app/ts/components/subcomponents/coins.tsx

```diff
@@ -218,6 +218,7 @@ type TokenWithAmountParams = TokenSymbolParams & {
 export function TokenWithAmount(param: TokenWithAmountParams) {
 	return <div style = 'width: fit-content; display: flex'>
 		<TokenAmount { ...param } />
+		<p>&nbsp;</p>
```

Comment:
Is this effectively the same as `<br/>`?

Reply:
no, its space (" "), <br> is line break